optuna/integration/lightgbm_tuner/__init__.py

```diff
@@ -25,24 +26,21 @@
 def train(*args: Any, **kwargs: Any) -> Any:
     """Wrapper of LightGBM Training API to tune hyperparameters.
 
-    It tunes important hyperparameters (e.g., `min_child_samples` and `feature_fraction`) in a
-    stepwise manner. Arguments and keyword arguments for `lightgbm.train()
+    It tunes important hyperparameters (e.g., ``min_child_samples`` and ``feature_fraction``) in a
+    stepwise manner. You use it by changing one import statement in your code. Just replace
+    ``import lightgbm as lgb`` with ``import optuna.integration.lightgbm as lgb``. See
+    `a simple example of LightGBM Tuner <https://github.com/optuna/optuna/blob/master/examples/lig
+    htgbm_tuner_simple.py>`_ which optimizes the validation log loss of cancer detection.
+
+    :func:`~optuna.integration.lightgbm.train` is a wrapper function of
+    :class:`~optuna.integration.lightgbm_tuner.LightGBMTuner`, and please use it if you want to
+    utilize advanced features such as suspending/resuming optimization and parallelization.
```

Comment:
I'd write the following. Might sound a bit more natural.
> ... a wrapper function of :class:`~optuna.integration.lightgbm_tuner.LightGBMTuner`. To use feature in Optuna such as suspended/resumed optimization and/or parallelization, refer to :class:`~optuna.integration.lightgbm_tuner.LightGBMTuner` instead of this function.

Reply:
Thanks! Applied: https://github.com/optuna/optuna/pull/1217/commits/15a425c7903a414c759b23dfe1e638018b597189